vars/main.yml

```diff
@@ -10,6 +10,6 @@ vips_required_libs:
 
 vips_sources_package: "vips-{{ vips_version }}.tar.gz"
 
-vips_sources_url: "https://github.com/jcupitt/libvips/releases/download/v{{ vips_version }}/{{ vips_sources_package }}"
+vips_sources_url: vips_sources_url: "https://github.com/libvips/libvips/releases/download/v{{ vips_version }}/{{ vips_sources_package }}"
```

Comment:
Hi @chadwilken! Thanks for your contribution. I'm seeing that `vips_sources_url` is duplicated. Could you please fix that and update the changelog with your contribution?

Thanks in advance

Reply:
Whoops not sure how I made that mistake, but both of those items have been addressed.